### PR TITLE
added  abbreviations

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user feedback issue #6849, so i added abbreviations of  **WW** , **GCC** and **DoD**

### CODE REVIEW

**Review by cichur — COMMENTED** [review]
HI,

Will you change the spelling of the word "Defence" to "Defense"? Thank you, Cindy

**Comment by RAJU2529** [comment]
@cichur . yes I will change, send suggest changes commit, so that I will accept

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/view-only-meeting-experience.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Attendees who join after the main meeting capacity has been reached will have a
 We have full Android and iOS mobile support for an attendee to join.
 
 > [!Note]
-> The current limit for the number of people who can chat and call in to a meeting is 300 in WW and 250 in GCC, GCC High, and DoD.
+> The current limit for the number of people who can chat and call in to a meeting is 300 in **WorldWide (WW)** and 250 in **Government Community Cloud (GCC)**, **Government Community Cloud (GCC) High**, and **Department of Defence (DoD)**.
```
Comment (JohanFreelancer9) [inline Teams/view-only-meeting-experience.md:38]:
```suggestion
> The current limit for the number of people who can chat and call into a meeting is 300 in WorldWide (WW) and 250 in Government Community Cloud (GCC), GCC High, and Department of Defense (DoD).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by cichur — CHANGES_REQUESTED** [review]
In Line 38, will you change "into" back to "in to"? Thank you.

File: Teams/view-only-meeting-experience.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Attendees who join after the main meeting capacity has been reached will have a
 We have full Android and iOS mobile support for an attendee to join.
 
 > [!Note]
-> The current limit for the number of people who can chat and call in to a meeting is 300 in **WorldWide (WW)** and 250 in **Government Community Cloud (GCC)**, **Government Community Cloud (GCC) High**, and **Department of Defence (DoD)**.
+> The current limit for the number of people who can chat and call into a meeting is 300 in WorldWide (WW) and 250 in Government Community Cloud (GCC), GCC High, and Department of Defense (DoD).
```
Comment (JohanFreelancer9) [inline Teams/view-only-meeting-experience.md:38]:
```suggestion
> The current limit for the number of people who can chat and call in to a meeting is 300 in WorldWide (WW) and 250 in Government Community Cloud (GCC), GCC High, and Department of Defense (DoD).
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by cichur** [comment]
Hi @JohanFreelancer9 , I pushed a newer version of this article yesterday. The content in the article that you edited is old. Therefore, I'll be closing your PR. Thank you, Cindy

**Pull request closed without merging**